db/async/row.go

```diff
@@ -78,7 +78,11 @@ func (a *Async) TaosExec(taosConnect unsafe.Pointer, sql string, timeFormat wrap
 					if row == nil {
 						return nil, FetchRowError
 					}
-					values[j] = wrapper.FetchRow(row, j, rowsHeader.ColTypes[j], lengths[j], precision, timeFormat)
+					v := wrapper.FetchRow(row, j, rowsHeader.ColTypes[j], lengths[j], precision, timeFormat)
+					if vv, is := v.([]byte); is {
+						v = string(vv)
```

Comment:
One question: how the json data value repr in json response? Raw string or JSON object ?

Reply:
I think it's reasonable to return JSON object. Fixed